travis/travis_install_nightly

```diff
@@ -2,11 +2,6 @@
 
 pip install -q QUnitSuite flake8 coveralls pylint > /dev/null 2>&1
 
-# We can exit here and do nothing if this only a LINT check
-if [ "${LINT_CHECK}" == "1" ] ; then
-    exit 0
-fi
```

Comment:
I see what you mean now. My mind was on another file.
Maybe we could:

    if [ "${TESTS}" == "0" ] ; then

@moylop260 can you fix this, to avoid a PR on your PR?